// GENERATED CODE - DO NOT EDIT
// Generated by schema2code proto generator

syntax = "proto3";

package proto;

option go_package = "maistro/proto";

// Response from generating an image
message ImageGenerateResponse {
  // Base64-encoded image
  string image = 1;
  // URL to download the image
  string download = 2;
}
